src/lib/rest/rest.cpp

```diff
@@ -616,6 +630,17 @@ static int contentTypeCheck(ConnectionInfo* ciP)
     return 1;
   }
 
+
+  // Case 5
+  if ((ciP->apiVersion == "v2") && (ciP->httpHeaders.contentType != "application/json"))
```

Comment:
What about the opossite? I mean, checking that the Accept includes "application/json" or "*/*" otherwise return a "Media not supported" (or similar... I don't remember exactly right now). I mean, from https://github.com/telefonicaid/fiware-orion/issues/947

> This operation only works with Accept "application/json". Attems of using other Accepts (inclusing application/xml) would result in an error.

By the way, I understand that if the request doesn't include an Accept header, the default Accept is "*/*" (from http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html):

> If no Accept header field is present, then it is assumed that the client accepts all media types